scripts/deploy_glastopf.sh

```diff
@@ -10,6 +10,17 @@ if [ $# -ne 2 ]
         exit 1
 fi
 
+# Check if Ubuntu 14.04 or 16.04
+if [ "$(lsb_release -r -s)" != "14.04" ] && [ "$(lsb_release -r -s)" != "16.04" ]; then
```

Comment:
Below we have conditional checks and install dependencies for 14.04 but nothing for 16.04.

1. Is 16.04 guaranteed to not work?
2. Does it not require the same dependencies as 14.04?
3. If no to Q1 and Q2, does 16.04 not require any additional install dependencies?

Reply:
So first we check to see if it is neither 14.04 or 16.04. If it's neither of these, we warn that it isn't supported. Ie. We support only 14.04 and 16.04 with this script.